bicepconfig.json

```diff
@@ -3,7 +3,7 @@
         "extensibility": true
     },
     "extensions": {
-        "radius": "br:biceptypes.azurecr.io/radius:latest",
-        "aws": "br:biceptypes.azurecr.io/aws:latest"
+        "radius": "br:biceptypes.azurecr.io/radius:0.39",
```

Comment:
this file shouldn't be updated but looks like it was overwritten. let's keep it as latest

Reply:
fixed